IDEOorg/steps
# #161240394 Disable autocomplete for new client form

#### What does this PR do?
Disable autocomplete on new client form for privacy reasons.

#### Description of Task to be completed?
- set autocomplete to off on new client form

#### How should this be manually tested?
- Sign in as a coach
- Navigate to `add a new client` page
- Interacting with any of the input fields should not trigger `autocomplete/suggestion`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#161240394

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
